localStorage.js

```diff
@@ -0,0 +1,26 @@
+const inputTitle = document.getElementById('title');
+const inputAuthor = document.getElementById('author');
+
+function storedData() {
+  const formData = {
+    title: inputTitle.value,
+    author: inputAuthor.value,
+  };
+  localStorage.setItem('form', JSON.stringify(formData));
+}
+
+function getData() {
+  const receivedData = localStorage.getItem('form');
+  if (receivedData) {
+    const formObject = JSON.parse(receivedData);
+    inputTitle.value = formObject.title;
+    inputAuthor.value = formObject.author;
+  }
+}
+
+window.onload = () => {
+  getData();
+};
+
+inputTitle.onchange = storedData;
+inputAuthor.onchange = storedData;
```

Comment:
- [ ] Kindly when I tried to add some books, the books didn't get saved on the local storage.

![image](https://user-images.githubusercontent.com/35535130/175986807-85af3470-dd2d-4538-8510-5da703172119.png)

Reply:
@keroloussamy We appreciate your feedback. We will like to confirm that the local storage is functional. 
Attached is an image of the local storage display.
![evidence](https://user-images.githubusercontent.com/92553079/176023323-baf9ca00-688b-4e0b-888c-3774db2e1368.png)